packages/office-ui-fabric-react/src/components/HoverCard/HoverCard.base.tsx

```diff
@@ -82,6 +83,18 @@ export class HoverCardBase extends BaseComponent<IHoverCardProps, IHoverCardStat
     }
   }
 
+  public dismiss = (withTimeOut?: boolean): void => {
+    this._async.clearTimeout(this._openTimerId);
+    this._async.clearTimeout(this._dismissTimerId);
+    if (!withTimeOut) {
+      this._setDismissedState();
+    } else {
+      this._dismissTimerId = this._async.setTimeout(() => {
+        this._setDismissedState();
+      }, this.props.cardDismissDelay!);
```

Comment:
Why the need for the `!`? Should this be used if it's not defined?

Reply:
This was from the very beginning... and yes it should be used because if it's not passed through props by the consumer we are setting it in `defaultProps`.